src/Env/Defaults/Defaults.php

```diff
@@ -9,27 +9,27 @@
  * file that was distributed with this source code.
  */
 
-namespace Manala\Manalize\Env\Metadata;
+namespace Manala\Manalize\Env\Defaults;
 
 /**
- * A metadata bag containing all defaults for a given env type.
+ * A bag containing all defaults for a given env.
  *
  * @author Robin Chalas <robin.chalas@gmail.com>
  */
-final class MetadataBag
+final class Defaults
```

Comment:
I'm not sure I like much this renaming.
What are your motivations behind this ?